EarlGrey/Action/GREYActions.m

```diff
@@ -262,6 +266,31 @@ @implementation GREYActions
 #pragma mark - Private
 
 /**
+ *  Set the UITextField text value directly, bypassing the iOS keyboard.
```

Comment:
It's possible but the goal is to avoid the keyboard, I think it's cleaner as a different action. I'd have to add a bunch of conditional logic to avoid the clear action and the `grey_actionForTypeText`. 

I noticed that `actionForClearText` supports webviews though and the current implementation of grey_replaceText doesn't.